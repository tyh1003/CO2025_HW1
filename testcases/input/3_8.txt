10
28 12 16 11 79 54 45 60 37 34
