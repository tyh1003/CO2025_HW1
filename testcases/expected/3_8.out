11 12 16 28 34 37 45 54 60 79 
